app/pages/task-list/neworder/ui.test.js

```diff
@@ -178,7 +178,7 @@ test('should render the third task as "Commencement date" and "data" as text not
     .expect(thirdTaskFirstItem.find('a').exists).notOk();
 });
 
-test('should render the fourth task as "Service recipients" and "data" as text not link', async (t) => {
+test('should render the fourth task as "Select Service Recipients" and "service recipients" as text not link', async (t) => {
```

Comment:
these test names don't make much sense

Reply:
🤔 I think I'll refactor these to the same style as what I've done for the existing order ui tests.